cocos/tween/tween.ts

```diff
@@ -127,7 +146,7 @@ export class Tween<T extends object = any> {
         if (this._finalAction) {
             TweenSystem.instance.ActionManager.removeAction(this._finalAction);
         }
-        this._finalAction = this._union();
+        this._finalAction = this._union(false);
```

Comment:
The final action doesn't need to update workerTarget since it will replace workerTarget (subTarget) with the `root target`.

Reply:
What does `workerTarget` mean?

Reply:
I answered it above, maybe you could refer it by scroll up this page.